examples/distro/build.gradle

```diff
@@ -0,0 +1,57 @@
+group 'io.opentelemetry.example'
+version '1.0-SNAPSHOT'
```

Comment:
This might be too much personal preference, but I wonder if it's worth demonstrating a versioning pattern where the distro name is suffixed? e.g., `1.0.0-aws.1`, `1.0.0-demo.1`. It's a free way of getting the `telemetry.sdk` stuff to identify the distro and might be nice for vendors (I'm not doing it yet but am planning on it).

Reply:
Take a look at `Implementation-Version` in `agent/build.gradle`. I totally agree with you on `telemetry.sdk`, but I think that jar built from the project should have just "local" version. Vendor name and otel version can go to manifest.